vllm/executor/ray_gpu_executor.py

```diff
@@ -430,7 +430,7 @@ def _compiled_ray_dag(self, enable_asyncio: bool):
         import pkg_resources
         from packaging import version
 
-        required_version = version.parse("2.32")
+        required_version = version.parse("2.35")
```

Comment:
I think it is not enough? It is possible users use 2.35 but ray[adag] is not downloaded

Reply:
This path is used in CI, and CI should be able to catch it?

Reply:
I am talking about more of end user perspective

Reply:
I think this is not sufficient error messgae. We should 1. detect ray[adag] is not downloaded & and raise an error message users should download `pip install ray[adag]`. With the current code, if users download `pip install ray==2.35`, it passes this part and will have weird dependency error.

Reply:
I think I tested with checking "ray[adag]" version last week and it did not make a difference, so I kept this. Let me look further.